company-profiles/nuharborsecurity.md

```diff
@@ -0,0 +1,36 @@
+# nuharborsecurity
```

Comment:
Company name should match that in the readme, so this should be `Nuharbor Security`

Reply:
Hello, I have adjusted the P.R. Please let me know if this is sufficient or still requires changes.
Thanks.